controllers/controllers.js

```diff
@@ -0,0 +1,10 @@
+const { fetchTopics } = require("../models/models.js");
+
+const getTopics = (req, res, next) => {
+    return fetchTopics().then((topics) => {
+        res.status(200).send(topics)
```

Comment:
anything from a controller needs to be sent back on a meaningful key ie. .send({topics})